src/stores/FrameStore.ts

```diff
@@ -435,17 +435,28 @@ export class FrameStore {
     }
 
     @computed get positionAxis(): number {
+        return this.specialAxis(/offset|position|offset position/i);
+    }
+
+    @computed get isUVImage(): boolean {
+        return this.uvAxis !== undefined;
+    }
+
+    @computed get uvAxis(): number {
+        return this.specialAxis(/uu/i);
+    }
+
+    private specialAxis = (regex): number => {
         if (this.frameInfo?.fileInfoExtended?.headerEntries) {
             const entries = this.frameInfo.fileInfoExtended.headerEntries;
-            const positionAxisRegex = /offset|position|offset position/i;
             const axis1 = entries.find(entry => entry.name.includes("CTYPE1"));
             const axis2 = entries.find(entry => entry.name.includes("CTYPE2"))
-            if (axis1?.value?.match(positionAxisRegex) || axis2?.value?.match(positionAxisRegex)) {
-                return axis1?.value?.match(positionAxisRegex) ? 1 : 2;
+            if (axis1?.value?.match(regex) || axis2?.value?.match(regex)) {
```

Comment:
personally I think this would be easier to understand as an `if (...) else if (...)` block.

Reply:
Changed to an `if ... else if` block.